pkg/provider/acceptance_source_postgres_test.go

```diff
@@ -36,10 +36,10 @@ func TestAccSourcePostgres_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("materialize_source_postgres.test", "size", "3xsmall"),
 					resource.TestCheckResourceAttr("materialize_source_postgres.test", "text_columns.#", "1"),
 					resource.TestCheckResourceAttr("materialize_source_postgres.test", "table.#", "2"),
-					resource.TestCheckResourceAttr("materialize_source_postgres.test", "table.0.name", "table1"),
-					resource.TestCheckResourceAttr("materialize_source_postgres.test", "table.0.alias", fmt.Sprintf(`%s_table1`, connName)),
-					resource.TestCheckResourceAttr("materialize_source_postgres.test", "table.1.name", "table2"),
-					resource.TestCheckResourceAttr("materialize_source_postgres.test", "table.1.alias", fmt.Sprintf(`%s_table2`, connName)),
+					// resource.TestCheckTypeSetElemAttr("materialize_source_postgres.test", "table.0.name.*", "table1"),
```

Comment:
`TestCheckTypeSetElemAttr` seems more appropriate (though this still isn't set quite right). Since switching this to `TypeSet`.